setup.py

```diff
@@ -9,8 +9,8 @@
 from setuptools import setup, find_packages
 
 
-if sys.hexversion < 0x3050000:
-    sys.exit("Error: MoinMoin requires Python 3.5+., current version is %s\n" % (platform.python_version(), ))
+if sys.hexversion < 0x3080000:
+    sys.exit("Error: MoinMoin requires Python 3.8+., current version is %s\n" % (platform.python_version(), ))
```

Comment:
this is quite old code (not possible otherwise many years ago). but since a while, there is a better method:

https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires